Exiled.API/Features/Round.cs

```diff
@@ -26,6 +26,11 @@ public static class Round
         /// </summary>
         public static DateTime StartedTime => DateTime.Now - ElapsedTime;
 
+        /// <summary>
+        /// Gets the round progress time.
+        /// </summary>
+        public static TimeSpan ProgressTime => DateTime.Now.Subtract(StartedTime);
+
```

Comment:
What does this value actually mean? `Now - (Now - ElapsedTime)` will just end up being `ElapsedTime`, so this property seems redundant.